src/shared/utilities/pathUtils.ts

```diff
@@ -48,10 +48,11 @@ export function normalizeSeparator(p: string) {
     return normalized
 }
 
-export function dirnameWithTrailingSlash(path: string): string {
-    let dirname = _path.dirname(path)
-    if (!dirname.endsWith(_path.sep)) {
-        dirname += _path.sep
+export function normalizedDirnameWithTrailingSlash(path: string): string {
```

Comment:
We can avoid long names by just explaining details in the docstring where necesary. Otherwise we end up explaining the function in the function name itself, which usually is just noise.

Reply:
I want to remove this function entirely, it's only used in one place and is not really that great. I hacked this name together because it fixes windows